addons/arsenal/functions/fnc_updateUniqueItemsList.sqf

```diff
@@ -191,7 +194,7 @@ private _itemInfoType = 0;
                         {!(_x in (GVAR(virtualItems) get IDX_VIRT_EXPLOSIVES))} &&
                         {!(_x in (GVAR(virtualItems) get IDX_VIRT_ITEMS_ALL))}
                     ) then {
-                        (GVAR(virtualItems) get IDX_VIRT_UNIQUE_UNKNOWN_ITEMS) set [_x, nil];
+                        (GVAR(virtualItems) get IDX_VIRT_UNIQUE_UNKNOWN_ITEMS) set [_x, true];
```

Comment:
This'll mark weapons, containers, and goggles as unique.

Reply:
Weapons and containers are fine to me, goggles not.

Reply:
> This condition'll mark weapons and goggles as unique.

There is an upper check for `_isMagazine`, so it never comes to this. However, goggles are still regarded as unique, so I need to find out why.

Reply:
Goggles are handled separately, that's why. Fixing.

Reply:
Disregard containers, bugged.

Reply:
Addressed by #9701.